### PR TITLE
[SpringBoard, BulletinBoard] Improvements

### PR DESCRIPTION
**Pull request by ryannair05**

### CODE REVIEW

File: BulletinBoard/BBBulletin.h [hunk 1]
```diff
@@ -25,22 +25,23 @@ typedef NS_ENUM(NSInteger, BBAttachmentMetadataType) {
 @property (nonatomic, copy) NSString *title;
 @property (nonatomic, copy) NSString *subtitle;
 @property (nonatomic, copy) NSString *message;
+@property (nonatomic, retain) BBContent * content;
```
Comment (lateef-class) [inline BulletinBoard/BBBulletin.h:28]: nitpick: space between `*` and variable name
```suggestion objc
@property (nonatomic, retain) BBContent *content;
```

Comment (leptos-null) [inline BulletinBoard/BBBulletin.h:28]: nitpick: space between `*` and variable name
```suggestion objc
@property (nonatomic, retain) BBContent *content;
```

File: SpringBoard/SBApplicationIcon.h [hunk 2]
```diff
@@ -2,8 +2,16 @@
 
 @class SBApplication;
 
-@interface SBApplicationIcon : SBLeafIcon
+typedef struct SBIconImageInfo {
+	CGSize size;
+	CGFloat scale;
+	CGFloat continuousCornerRadius;
+} SBIconImageInfo;
```
Comment (leptos-null) [inline SpringBoard/SBApplicationIcon.h:9]: This struct exists in [SpringBoard/SBIconImageInfo.h](https://github.com/theos/headers/blob/819f49385304805f9a81e4210ae392c620505ea8/SpringBoard/SBIconImageInfo.h#L3-L7)

**Comment by leptos-null** [comment]
Thank you for adding availability attributes! Looks great

**Review by leptos-null — APPROVED** [review]
Looks great- thanks!

File: SpringBoard/SBLockScreenManager.h [hunk 3]
```diff
@@ -1,13 +1,14 @@
...
+API_AVAILABLE(ios(7.0)) @interface SBLockScreenManager : NSObject
 
 + (instancetype)sharedInstance;
 
 - (void)lockUIFromSource:(NSUInteger)source withOptions:(NSDictionary *)options;
 - (void)unlockUIFromSource:(NSUInteger)source withOptions:(NSDictionary *)options;
 
-@property (nonatomic, retain) SBLockScreenViewController *lockScreenViewController;
+@property (nonatomic, retain) SBLockScreenViewController *lockScreenViewController API_DEPRECATED_WITH_REPLACEMENT("coverSheetViewController", ios(7.0, 13.0));
+@property (nonatomic,readonly) CSCoverSheetViewController *coverSheetViewController API_AVAILABLE(ios(13.0));
```
Comment (PoomSmart) [inline SpringBoard/SBLockScreenManager.h:11]:
```suggestion
@property (nonatomic, readonly) CSCoverSheetViewController *coverSheetViewController API_AVAILABLE(ios(13.0));
```

**Review by PoomSmart — CHANGES_REQUESTED** [review]

**Review by PoomSmart — APPROVED** [review]

**Pull request merged**